docs/Docker.md

```diff
@@ -20,11 +20,12 @@ The following steps can be used to setup Stringer on Docker, using a Postgres da
 docker run --detach \
     --name stringer-postgres \
     --restart always \
-    --volume /srv/stringer/data:/var/lib/postgresql/data \
+    --volume ./postgres:/var/lib/postgresql/data \
     --net stringer \
-    -e POSTGRES_PASSWORD=myPassword \
+    -e POSTGRES_PASSWORD=<PLEASE_ENTER_YOUR_PASSWORD> \
+    -e POSTGRES_USER=db_user \
     -e POSTGRES_DB=stringer \
-    postgres:9.5-alpine
+    postgres:12-alpine
```

Comment:
How come you chose version `12` instead of the most recent `15`?

Reply:
I had an issues with postgres 9 which update all my folder permissions. Apparently it was fixed in 12 version, and it's the first version with the fix. May be you can use 15 version, but I haven't try it. 

I could do some test if you want !

Reply:
@Ni-g-3l sure, that would be helpful! I'm running 15.3 with my instance. It's not a huge deal, though, we can always upgrade more later.

Reply:
I just upgraded my instance, let's pass this week and see what will happen.

Reply:
First issues incomming, you can't point to same folder postgres12 and postgres15 
![image](https://github.com/stringer-rss/stringer/assets/42718803/1a3a8fc5-d9f9-45e9-a3df-8230bd393371)

Reply:
I think this is more because of an upgrade of postgres using the same data directory, rather than a problem with postgres 15 itself. So updating from 9 to 12 is already going to give issues, I think we may as well do 15.

I think this is probably where a stringer export comes in - we could export the data, upgrade to a newer version of postgres, and then re-import.

imo, we should set this to 15. If a new user comes to stringer, they'll want the latest version of postgres; if a previous user is updating, they should know that they need to back up their system and that changing postgres versions incurs problems. We may want to announce this in some way in the next release though, as a PSA.

Reply:
(also, fwiw, someone else I know from the FOSS community created [this repo](https://github.com/pgautoupgrade/docker-pgautoupgrade) that auto-upgrades postgres without issues like this, and then runs the container as postgres normally would. But it's an additional layer of complexity, and I imagine postgres will eventually coalesce things into their docker image)

Reply:
Hmm, I'm not sure what the best approach here is. I'm imagining the upgrade process would be smoother if it was more gradual, but that seems like a pretty lengthy process. That repo you linked could be really handy to smooth that process.

> We may want to announce this in some way in the next release though, as a PSA.

I unfortunately haven't figured out a good release process yet. Maybe we need to set up a changelog and tagging somehow. It's a little weird, though, since we aren't releasing a gem or the like.